pkgs/development/python-modules/devtools/default.nix

```diff
@@ -0,0 +1,22 @@
+{ buildPythonPackage, fetchPypi, lib, pygments }:
+buildPythonPackage rec {
+  pname = "devtools";
+  version = "0.6.1";
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1ms2kp1h395880k1bcn71ap53s33ffb2jxnzx3x2hpfkjisk0m50";
+  };
+
+  propagatedBuildInputs = [ pygments ];
+
+  # tests not distributed on pypi
+  doCheck = false;
```

Comment:
There are GitHub releases present. Please use those and run the [tests](https://github.com/samuelcolvin/python-devtools/tree/master/tests).